source/fundamentals/serialization/poco.txt

```diff
@@ -207,22 +255,334 @@ Omit Empty Fields
+
+.. code-block:: csharp
+   :copyable: false
+   :emphasize-lines: 3
+   
+   public class House
+   {
+       [BsonId(IdGenerator = typeof(NullIdChecker))]
+       public Guid Id { get; set; }
+   }
+
+Customize DateTime Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To customize how the {+driver-short+} serializes ``DateTime`` properties, use the
+``[BsonDateTimeOptions()]`` attribute and specify the desired setting as an argument.
+
+If a ``DateTime`` property represents only a date, you can apply the
+``[BsonDateTimeOptions(DateOnly = true)]`` attribute to it.
+If you do, the driver won't perform any time-zone conversion on the value.
+
+In the following code example, the ``PatientRecord`` class uses a ``DateTime`` for the
+``DateOfBirth`` property. The ``[BsonDateTimeOptions(DateOnly = true)]``
+attribute indicates that the property contains only a date.
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 5
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(DateOnly = true)]
+       public DateTime DateOfBirth { get; set; }
+   }
+
+You can also use the ``[BsonDateTimeOptions()]`` attribute to specify the ``DateTimeKind``
+of a ``DateTime`` property. In the following code example, the ``PatientRecord`` class
+uses a ``DateTime`` for the ``AppointmentTime`` property.
+The ``[BsonDateTimeOptions(Kind = DateTimeKind.Local)]`` attribute indicates
+that the time component of the property's value is in local time. When the
+driver serializes this property, it converts the time to UTC, the standard
+format for times stored in MongoDB.
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 8
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(DateOnly = true)]
+       public DateTime DateOfBirth { get; set; }
+
+       [BsonDateTimeOptions(Kind = DateTimeKind.Local)]
+       public DateTime AppointmentTime { get; set; }
+   }
+
+You can also specify one or both of the previous ``DateTime`` options when
+registering the class map:
+
+.. code-block:: csharp
+   :copyable: true
+
+   BsonClassMap.RegisterClassMap<House>(classMap => 
+   {
+      classMap.AutoMap();
+      classMap.MapMember(p => p.DateOfBirth).SetSerializer(new DateTimeSerializer(dateOnly: true));
+      classMap.MapMember(p => p.AppointmentTime).SetSerializer(new DateTimeSerializer(DateTimeKind.Local));
+   });
+
+.. tip:: DateTimeKind Values
+
+   The ``DateTimeKind`` enum is part of the {+framework+}. For more information on
+   its members, see the `Microsoft documentation for the DateTimeKind enum. <https://learn.microsoft.com/en-us/dotnet/api/system.datetimekind?view=net-8.0>`__
+
+Customize Dictionary Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``DictionaryRepresentation`` enum defines the formats the {+driver-short+}
+can serialize a ``Dictionary`` to. This enum includes the following members:
```

Comment:
```suggestion
The ``DictionaryRepresentation`` enum defines the target format that the {+driver-short+} serializes
``Dictionary`` instances into. This enum includes the following members:
```

Reply:
I was describing the enum variable itself, which contains three different possible formats.

Added `instances`